src/internal/connector/graph/service.go

```diff
@@ -9,9 +9,23 @@ import (
 	"github.com/alcionai/corso/src/pkg/path"
 )
 
-// DeltaTokenFileName is the name of the file containing delta token(s) for a
-// given endpoint. The endpoint granularity varies by service.
-const DeltaTokenFileName = "delta"
+const (
+	// DeltaTokenFileName is the name of the file containing delta token(s) for a
+	// given endpoint. The endpoint granularity varies by service.
+	DeltaTokenFileName = "delta"
+	// PreviousPathFileName is the name of the file containing previous path(s) for a
+	// given endpoint.
+	PreviousPathFileName = "previouspath"
+)
+
+const (
+	// DeltaMetadataCollectionKey is the name of the collection holding the delta
+	// url lookups inside a collection map otherwise keyed by folder ids.
+	DeltaMetadataCollectionKey = "delta-metadata"
+	// PathsMetadataCollectionKey is the name of the collection holding the previous
+	// paths lookups inside a collection map otherwise keyed by folder ids.
+	PathsMetadataCollectionKey = "paths-metadata"
```

Comment:
this is just an artifact of needing to have some map key so we can add it to the set of collections the exchange code deals with. It could really be a single map key (since we should only be making one collection) and it should be in the exchange files not here (it's not something that will be needed outside of backup). Though, it may also be possible to use the `col.FullPath().ShortRef()` or `col.FullPath().String()` as the map key for the metadata collection

The map was used just to make sure that we only had one instance of a collection for each path when we were fetching items in a different way. It's probably not really required anymore but that code hasn't been refactored in a while